src/components/TopNav/TopNav.tsx

```diff
@@ -21,7 +21,7 @@ const StyledMenuItem = styled(MenuItem)({
 })
 
 const TopNav = () => {
-  const { user } = useAuth()
+  const { user } = useAuth() // TODO this should pull from the profile instead
```

Comment:
we don't need to pull from profile. Since `useAuth()` is a hook that inherently uses `useSWR`, all network requests are cached, so it won't send requests twice

Reply:
yeah that comment is out of date